prov/verbs/src/fi_verbs.h

```diff
@@ -199,6 +199,11 @@ extern struct fi_ibv_gl_data {
 	} dgram;
 } fi_ibv_gl_data;
 
+struct fi_ibv_verbs_prov_preset {
+	int		max_inline_data;
+	const char	*dev_name_prefix;
+};
+
```

Comment:
Can we rename this to fi_ibv_dev_preset since these are device specific?

Reply:
This definition can be moved into fi_verbs.c, as it is only used there.

Reply:
Ok, both comments have been applied